lib/package.json

```diff
@@ -16,7 +16,10 @@
   "peerDependencies": {
     "react": ">=16.9",
     "react-native": ">=0.58",
-    "emotion": ">=11.0.0"
+    "emotion": ">=11.0.0",
+    "@emotion/react": "^11.4.1",
+    "@emotion/native": "^11.0.0",
+    "react-native-vector-icons": "^8.1.0"
```

Comment:
How about adding more scopes to the versions like ">=11.0.0"?

Reply:
For `react-native-vector-icons`, specifying it like below would be great.

```
"react-native-vector-icons": "*"
```

Reply:
ah, thank you for version checking. I didn't consider that. I will fix soon. 👏

Reply:
@hyochan
I fixed it. please check PR 🙇